pkg/chunk/storage/factory.go

```diff
@@ -117,19 +118,20 @@ func (cfg *Config) Validate() error {
 func NewStore(cfg Config, storeCfg chunk.StoreConfig, schemaCfg chunk.SchemaConfig, limits StoreLimits, reg prometheus.Registerer, cacheGenNumLoader chunk.CacheGenNumLoader) (chunk.Store, error) {
 	chunkMetrics := newChunkClientMetrics(reg)
 
-	indexReadCache, err := cache.New(cfg.IndexQueriesCacheConfig)
+	logger := log.NewNopLogger()
```

Comment:
Uhm, no, we should pick it in input, not using the nop logger :)

Reply:
Thanks, yes what I am doing here is totally wrong :(.

But the logger is initialized in the service module, I need to do some work to propagate that logger here